kolibri/core/auth/tasks.py

```diff
@@ -241,7 +241,7 @@ def validate(self, data):
             facility_name = facility.name
         else:
             facility_id = facility
-            facility_name = data["facility_name"]
```

Comment:
This was put here to require the frontend to send the facility name to the backend, so that we can display it properly to the user during progress tracking - I think this must be available on the frontend to send.